qiskit_ibm_runtime/runtime_job.py

```diff
@@ -671,3 +673,64 @@ def usage_estimation(self) -> Dict[str, Any]:
             }
 
         return self._usage_estimation
+
+    def queue_position(self, refresh: bool = False) -> Optional[int]:
+        """Return the position of the job in the server queue.
+
+        Note:
+            The position returned is within the scope of the provider
+            and may differ from the global queue position.
+
+        Args:
+            refresh: If ``True``, re-query the server to get the latest value.
+                Otherwise return the cached value.
+
+        Returns:
+            Position in the queue or ``None`` if position is unknown or not applicable.
+        """
+        if refresh:
+            api_metadata = self._api_client.job_metadata(self.job_id())
+            self._queue_info = QueueInfo(
+                position_in_queue=api_metadata.get("position_in_queue"),
+                status=self._api_client.backend_status(self._backend.name),
```

Comment:
I don't think the status here is the backend status but instead the job status

Reply:
Changed to `job.status()`.